### ISSUE TITLE
Remove the ending zeros from exposed pts values

### ISSUE THREAD

**Issue by chros73** [cpm-code/xbmc#46]
Hi, can you remove the  ending zeros from [exposed pts values](https://github.com/cpm-code/xbmc/commit/94b4166fba04dffda6b1641ba6946187177daa92)? I guess they represent ms. 
E.g. diff value: -234000.000000 ->-234000
Thanks

**Comment by cpm-code** [comment]
Yep can remove the trailing zeros, pts are type double in kodi for some reason.

number is in DVD_TIME_BASE of kodi: (1000000)

So divide by that to get seconds and then fractions, above example: -0.234
From what I recall seeing it is normally rounded at the ms.

Those exposed values are there for my testing, was just looking for a offset between the audio and video pts to try and understand what is going on, an picked a consistent point for audio and video to work out a consistent diff overtime, otherwise those pts likely are not a match for the output as did not need to be for this purpose.

Also the audio pts is only on the passthrough flow.

FYI: Latest WIP now tracks the video pts much closer to the dvd clock in the RenderManager so no need to work this out now, looking to put out a new version when have a few other things worked out.

**Comment by chros73** [comment]
> Those exposed values are there for my testing, was just looking for a offset between the audio and video pts to try and understand what is going on

Thought so.

> otherwise those pts likely are not a match for the output as did not need to be for this purpose.

I wanted to expose the AV diff value, because I thought it can be relevant for the user.
But if you say it's not accurate/important then you can just leave it as it is and close this issue.

And thanks for trying to work out the audio sync issue!

**Comment by cpm-code** [comment]
Close - may expose RenderPts in future as is accurate to what is displaying.
